challenge.md

```diff
@@ -6,19 +6,27 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+'git clone (SSH link)'' to clone the repo into my computer
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+'git diff' will show you all the changes since last commit, but not all the changes are necessarily followed unless 'git add (folder or file)' to move changes into the staging area.
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+'git log'
+
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
 
+We want to create a new branch for that project and select that branch by using 'git checkout -b (new branch name)'
 
 5. I want to add a file to the staging area. What git commit would I use?
 
+'git add (files or folders to add to the staging area)'
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
 
+'git merge' to merge the master branch and your new branch
```

Comment:
Almost. You need to actually specify the name of the branch that you're merging in (in this case, it would be `git merge add-directories`). Also, `git merge` always merges towards where you are, so if you want to merge into `master`, you'll need to already be on that branch.